src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
 [assembly: AssemblyVersion("19.5.0.0")]
```

Comment:
Please set `AssemblyVersion` to `19.0.0.0`
`AssemblyFileVersion` is correct

Reply:
can we easily downgrade this version?

Reply:
Correcting myself, please leave `AssemblyVersion` unchanged

Reply:
@dsgouda  so can you please merge sdk?